src/WebJobs.Script/CloudBlockBlobHelperService.cs

```diff
@@ -25,18 +28,12 @@ private static async Task<bool> BlobExistsAsync(string url, string environmentVa
                     CloudBlockBlob blob = new CloudBlockBlob(new Uri(url));
                     exists = await blob.ExistsAsync();
                 }
-                catch (Exception e)
+                catch (Exception)
```

Comment:
We should probably improve this logic. As it stands, if the URL is invalid or doesn't point to a blob, we'll still do retries (e.g. if new Uri(url) throws). Instead, I think the intent is to retry only for transient errors arising from the Existence check. So we should only do retries if the latter part throws.

Reply:
Good catch, updating.